src/Servers/Kestrel/Transport.Quic/test/QuicConnectionContextTests.cs

```diff
@@ -551,6 +552,45 @@ public async Task PersistentState_StreamsReused_StatePersisted()
             Assert.Equal(true, state);
         }
 
+        [ConditionalFact]
+        [MsQuicSupported]
+        public async Task CertificatePassedToQuicConnectionContext()
```

Comment:
```suggestion
        public async Task TlsConnectionFeature_ClientSendsCertificate_PopulatedOnFeature()
```

Test name pattern: WhatIsBeingTested_Scenario_WhatHappened